job/tensorflowtrainingjob/multiworker/keras_mnist_trainingjob.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import json
+import os
+import shutil
+
+import tensorflow as tf
+from tensorflow.keras import callbacks, datasets, layers, models, optimizers
+
+parser = argparse.ArgumentParser(
+    description='Distributed training of Keras model for MNIST with '
+    'MultiWorkerMirroredStrategy.')
+parser.add_argument('--log_dir',
+                    type=str,
+                    default='/mnt/log',
+                    help='Path of the TensorBoard log directory.')
+parser.add_argument('--no_cuda',
+                    action='store_true',
+                    default=False,
+                    help='Disable CUDA training.')
+args = parser.parse_args()
+
+if args.no_cuda:
+    tf.config.set_visible_devices([], 'GPU')
+
+strategy = tf.distribute.MultiWorkerMirroredStrategy()
+
+# Get world size and index of current worker.
+tf_config = json.loads(os.environ['TF_CONFIG'])
+world_size = len(tf_config['cluster']['worker'])
+task_index = tf_config['task']['index']
+
+with strategy.scope():
+    model = models.Sequential([
+        layers.Conv2D(32, 3, activation='relu', input_shape=(28, 28, 1)),
+        layers.MaxPool2D((2, 2)),
+        layers.Conv2D(64, 3, activation='relu'),
+        layers.MaxPool2D((2, 2)),
+        layers.Conv2D(64, 3, activation='relu'),
+        layers.Flatten(),
+        layers.Dense(64, activation='relu'),
+        layers.Dense(10, activation='softmax'),
+    ])
+    model.compile(optimizer=optimizers.Adam(learning_rate=0.001 * world_size),
+                  loss='sparse_categorical_crossentropy',
+                  metrics=['accuracy'])
+
+(train_images, train_labels), (test_images,
+                               test_labels) = datasets.mnist.load_data(
+                                   path=os.path.join(os.getcwd(), 'mnist.npz'))
```

Comment:
It's not using the pre-downloaded dataset by `job/tensorflowtrainingjob/multiworker/download_dataset.py`

Reply:
fixed